topics/ios/index.md

```diff
@@ -9,4 +9,4 @@ topic: ios
 url: https://www.apple.com/ios/
 wikipedia_url: https://en.wikipedia.org/wiki/IOS
 ---
-iOS is the operating system for all of Apple’s mobile products. The operating system was unveiled at Macworld Conference and Expo in 2007 to support the company’s new venture, the iPhone. Since then, the operating system has grown to incorporate other products, including the iPad and iPod Touch.
+iOS is the   system for all of Apple’s mobile products. The operating system was unveiled at Macworld Conference and Expo in 2007 to support the company’s new venture, the iPhone. Since then, the operating system has grown to incorporate other products, including the iPad and iPod Touch.
```

Comment:
You only removed the word `operating` which should still go there.

Reply:
Cool